src/components/EditEvent/EventInformation.tsx

```
@@ -18,10 +18,11 @@ import { color, space, typography } from "wowds-tokens";
 import Button from "wowds-ui/Button";
 import DropDown from "wowds-ui/DropDown";
 import DropDownOption from "wowds-ui/DropDownOption";
-import { CopyUrlModal } from "./Modal/CopyUrlModal";
 
+import RoutePath from "@/routes/routePath";
```

ESLint

[error] `@/routes/routePath` import should occur before import of `@/types/dtos/event`

(import/order)